README.md

```diff
@@ -1,125 +1,82 @@
 
-## Install Kata containers runtime
+### Operator Architecture
 
-### Create the `KataConfig` CR to start the installation
-  
-``` 
-oc apply -f https://raw.githubusercontent.com/openshift/sandboxed-containers-operator/master/config/samples/kataconfiguration_v1_kataconfig.yaml
-```
+The following diagram shows how the operator components are connected to the OpenShift overall architecture:
 
-Please follow [this](#selectively-install-the-kata-runtime-on-specific-workers) section if you wish to install the Kata Runtime only on selected worker nodes.
+![High Level Overview](./docs/arch.png)
 
-#### Monitoring the Kata Runtime Installation
-Watch the description of the `KataConfig` custom resource
-```
-oc describe kataconfig example-kataconfig
-```
-and look at the field `Completed nodes` in the status. If the value matches the number of worker nodes the installation is completed.
 
-#### Runtime Class
-Once the sandboxed-containers extension is enabled successfully on the intended workers, the sandboxed containers operator will create a [runtime class](https://kubernetes.io/docs/concepts/containers/runtime-class/) `kata`. This runtime class can be used to deploy the pods that will use the Kata Runtime.
+Here is a brief summary of the components:
 
-#### Run an Example Pod using the Kata Runtime
-```
-oc apply -f https://raw.githubusercontent.com/openshift/sandboxed-containers-operator/master/config/samples/example-fedora.yaml
-```  
+- OpenShift clusters consist of controller and worker nodes organized as  machine config pools. 
+- The Machine Config Operator (MCO) manages the operating system and keeps the cluster up to date and configured.
+- The control-plane nodes run all the services that are required to control the cluster such as the API server, etcd, controller-manager, and the scheduler. 
+- The OpenShift sandboxed containers operator runs on a control plane node.
+- The cluster worker nodes run all the end-user workloads. 
+- The container engine `CRI-O` uses either the default container runtime `runc` or, in sandboxed containers case, the `Kata` containers runtime.
 
-## Selectively Install the Kata Runtime on Specific Workers
+### KataConfig Custom Resource Definition
 
-### Edit the custom resource file `config/samples/kataconfiguration_v1_kataconfig.yaml`
-   and uncomment the kata pool selector fields in the spec as follows,
+The operator owns and control the `KataConfig` Custom Resource Definition (CRD).
+Please refer to the [code](https://github.com/openshift/sandboxed-containers-operator/blob/master/api/v1/kataconfig_types.go) to find details of the `KataConfig` CRD. 
 
-   ```yaml
-   apiVersion: kataconfiguration.openshift.io/v1alpha1
-   kind: KataConfig
-   metadata:
-     name: example-kataconfig
-   spec:
-     kataConfigPoolSelector:
-       matchLabels:
-          custom-kata1: test
-   ```
+### Getting Started
```

Comment:
Should we reference https://docs.openshift.com/container-platform/4.9/sandboxed_containers/deploying-sandboxed-container-workloads.html somewhere here?

Reply:
Added in "Further Reading" section